src/components/AttachmentModal.js

```diff
@@ -393,17 +394,17 @@ function AttachmentModal(props) {
         }
         return menuItems;
         // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [isAttachmentReceipt, props.parentReport, props.parentReportActions, props.policy, props.transaction, file]);
+    }, [props.isReceiptAttachment, props.parentReport, props.parentReportActions, props.policy, props.transaction, file]);
 
     // There are a few things that shouldn't be set until we absolutely know if the file is a receipt or an attachment.
-    // isAttachmentReceipt will be null until its certain what the file is, in which case it will then be true|false.
+    // props.isReceiptAttachment will be null until its certain what the file is, in which case it will then be true|false.
     let headerTitle = props.headerTitle;
     let shouldShowDownloadButton = false;
     let shouldShowThreeDotsButton = false;
-    if (!_.isNull(isAttachmentReceipt)) {
-        headerTitle = translate(isAttachmentReceipt ? 'common.receipt' : 'common.attachment');
-        shouldShowDownloadButton = props.allowDownload && isDownloadButtonReadyToBeShown && !isAttachmentReceipt && !isOffline;
-        shouldShowThreeDotsButton = isAttachmentReceipt && isModalOpen;
+    if (!_.isEmpty(props.report)) {
```

Comment:
Seems this is causing an issue https://github.com/Expensify/App/issues/60092 of not showing the title or three dots button for a new user